src/main/Main.hs

```diff
@@ -414,24 +412,24 @@ commandLineHandler currentDir progName isInterpreter = complicatedOptions
       where
         -- addCommand hiding global options
         addCommand' :: String -> String -> (a -> RIO Runner ()) -> Parser a
-                    -> AddCommand
+                    -> AddCommand Runner
```

Comment:
If `env` is always `Runner`, this could revert to being just `AddCommand`, with a suitable type synonym.